x-pack/plugins/apm/e2e/yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Why do we have that yarn.lock file here? We are now in a single top level package.json so we should not have more than one yarn.lock file

Reply:
I can prevent it from being generated by removing this line: https://github.com/elastic/kibana/blob/b57651346c5685d7db9e594a0b1c882385dac8fd/x-pack/plugins/apm/e2e/run-e2e.sh#L101 (`yarn &> ${TMP_DIR}/e2e-yarn.log`) which was supposed to install dependencies before the last update. It's not needed anymore.

Reply:
Yeah let's do it 👍